### DISCUSSION TITLE
API for downloading the billing usage report

### DISCUSSION THREAD

**Discussion by chenrui333** [community/community#26941] [Actions]
<p>I saw the new of <a href="https://github.blog/changelog/2020-05-08-billing-usage-report-is-now-available-to-download/" rel="nofollow noopener">Billing usage report is now available to download</a>, I wonder if there is currently API that we can perodically download so that we can track our usage.</p>
<p>Thanks.</p>

**Comment by mscoutermarsh** [comment]


**Reply by VinodNallavade** [reply]
@mscoutermarsh Any update on this. If the API is already in place could you please share the link.

**Reply by risingh-apptio** [reply]
@mscoutermarsh ++ even I am looking for an API to download the CSV for usage report.

**Comment by jakubriegel** [comment]
@chenrui333 @risingh-apptio do you know is there is any API for that now?

**Comment by vishnu-valiveti** [comment]
Please any the API to download the custom report for billing, if available.

**Comment by nebuk89** [comment]
We are working on a new billing system that will in turn have an API :) and we will also put an API as the next thing we work on for the usage metrics section of Actions (behind the insights tab <3) movement coming here I hope in the near future

**Reply by ScriptAutomate** [reply]
Ooooo very interested in this! I'm right now needing to manually generate the CSV report with a scheduled reminder that I will definitely be automating when this is made available
